toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -2440,7 +2440,9 @@ void CellArea::mousePressEvent(QMouseEvent *event) {
       int c       = notes->getNoteCol(i);
       QPoint xy   = m_viewer->positionToXY(CellPosition(r, c));
       TPointD pos = notes->getNotePos(i) + TPointD(xy.x(), xy.y());
-      QRect rect(pos.x, pos.y, NoteWidth, NoteHeight);
+      QRect rect(pos.x, pos.y,
+                 (!o->isVerticalTimeline() ? NoteWidthTimeline : NoteWidth),
+                 NoteHeight);
```

Comment:
It works OK, but how about rewriting this line as 
`QRect rect = o->rect(PredefinedRect::NOTE_ICON).translated(pos.x, pos.y);` 
instead of defining the new constant values in order to make future maintenance easier?

Reply:
Heh.  I helped introduce those for easier maintenance and yet I still forgot to use it. :)  Thanks for the suggestion.  Will make the changes.